Makefile

```diff
@@ -106,7 +100,7 @@ docker_create:
 		-e GOOGLE_APPLICATION_CREDENTIALS=${CREDENTIALS_PATH} \
 		-v $(CURDIR):/cft/workdir \
 		${DOCKER_REPO_BASE_KITCHEN_TERRAFORM} \
-		/bin/bash -c "source test/ci_integration.sh && setup_environment && kitchen create"
+		/bin/bash -c "cd /cft/workdir && source test/ci_integration.sh && setup_environment && kitchen create"
```

Comment:
the need of change here and below is under similar question @kopachevsky raised [above](https://github.com/GoogleCloudPlatform/terraform-google-sql-db/pull/53/files#r310644882)

Reply:
Done